data_structures/delete_node_without_head.cpp

```diff
@@ -0,0 +1,108 @@
+/**
+ * \file
+ * \brief [Delete a Node Without Head Pointer
+ * in Linked List](https://en.wikipedia.org/wiki/Linked_list#Singly_linked_list)
+ * @author [Anshu Saini](https://github.com/anshu189)
+ */
+
+#include <iostream>
+using namespace std;
+
+// Initializing a Linked List using Class Node
+class Node{
+public:
+    int data;
+    Node *next;
+    // Constructor for Linked List for a deafult value
+    Node(int data){
+        this -> data=data;
+        this -> next=NULL;
+    }
+};
+
+// Print Linked List Function
+void printList(Node* &head){

```

Comment:
Functions should be documented explaining the use of its parameters, a brief description, and what it returns.